### DISCUSSION TITLE
Does PuLID work with Flux 1.0 [Schnell]?

### DISCUSSION THREAD

**Discussion by gedasv** [ToTheBeginning/PuLID#70] [General]
The question is the title.
Does PuLID work with Flux 1.0 [Schnell]? or is this only for [Dev]?
If not, is it possible to make it run on Schnell by modifying some code?

**Comment by lksenel** [comment]
I have tested it with the 8 bit quantized version of [Flux 1.0 [Schnell]](https://huggingface.co/Kijai/flux-fp8/tree/main) on [comfyUI](https://github.com/cubiq/PuLID_ComfyUI) and it seem to produce pretty good results (definitely much better than all the FaceID methods I tried before).  However, if you main reason to use Schnell version over the Dev version is to avoid non-commercial license of the Dev model, unfortunately it doesn't seem to help since PuLID seem to use [InsightFace ](https://github.com/deepinsight/insightface) models for the face analysis which also has non-commercial license. I hope this helps
